tests/Drupal/Tests/Driver/Kernel/CoreUserMethodsKernelTest.php

```
@@ -0,0 +1,108 @@
+ * Kernel test for user-related methods on Core.
+ *
+ * The whole lifecycle runs inside one test method on purpose: KernelTestBase's
+ * setUp runs per-method and costs roughly a second of bootstrap. Bundling
+ * closely related assertions here keeps CI time down without sacrificing
+ * coverage. Split a method out only when a scenario genuinely needs its own
+ * clean state (e.g. asserting a failure path that leaves the container dirty).
+ *
+ * To actually run this test, see the bootstrap/env notes in
+ * DatetimeHandlerKernelTest.
+ */
+class CoreUserMethodsKernelTest extends KernelTestBase {
```

PHPStan (2.1.46)

[error] Class Drupal\Tests\Driver\Kernel\CoreUserMethodsKernelTest extends unknown class Drupal\KernelTests\KernelTestBase.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(class.notFound)